### DISCUSSION TITLE
How to get that prompt when you run winget for the first time to come back?

### DISCUSSION THREAD

**Discussion by onyx-nxt** [microsoft/winget-cli#2751] [Q&A] [answered]
How do you get that prompt when you run winget for the first time to come back? It says about some agreement and requires a Y or N response. How would I see that message again?

**Comment by mdanish-kh** [comment, marked as answer]
The prompt is about Microsoft Store requiring you to accept source agreements. You can get the initial behavior by performing a source reset in an administrative terminal.

```pwsh
winget source reset --force
```
